django-stubs/test/selenium.pyi

```diff
@@ -11,8 +15,35 @@ class SeleniumTestCaseBase:
     def import_webdriver(cls, browser: Any) -> type[Any]: ...  # Type[WebDriver]
     def create_webdriver(self) -> Any: ...  # WebDriver
 
+class ChangeWindowSize:
+    def __init__(self, width: int, height: int, selenium: Any) -> None: ...
+    def __enter__(self) -> Self: ...
+    def __exit__(
+        self,
+        exc_type: type[BaseException] | None,
+        exc_value: BaseException | None,
+        traceback: TracebackType | None,
+    ) -> None: ...
+
 class SeleniumTestCase(LiveServerTestCase):
     implicit_wait: int
+    screenshots: bool = False
     selenium: Any
     @contextmanager
```

Comment:
In `typeshed` we try not to use `@contextmanager` decorator and instead use `-> contextlib.AbstractContextManager[...]`

Reply:
Ah, thanks, TIL. Found the docs: https://typing.python.org/en/latest/guides/writing_stubs.html#decorators